server/game/cards/01-Core/TirelessCrocag.js

```diff
@@ -6,19 +6,22 @@ class TirelessCrocag extends Card {
             condition: context => context.player.opponent.creaturesInPlay.length === 0,
```

Comment:
If you've got terminal conditions to work, this play ability shouldn't be here

Reply:
Correct. I will remove it.